src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -112,14 +119,14 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow one single parameterless annotation on the same line
+ * Example to allow multiple annotations on the same line:
  * </p>
  * <pre>
- * &#64;Override public int hashCode() { ... } // no violations
- * &#64;SuppressWarnings("deprecation") public int foo() { ... } // violation
+ * &#64;SuppressWarnings("deprecation") &#64;Mock DataLoader loader; // ok
```

Comment:
please provide CLI prove that it works like this in PR description.
Issue description has a link to example how PR should looks like - please do.

please keep in each example code with violations and without.

Reply:
Done